### PR TITLE
Make logs more helpful for E2E

### PR DESCRIPTION
**Pull request by 0xKiwi**
This adds a bit more detail to E2E logs on some evaluators, and standard errors.

### CODE REVIEW

File: endtoend/beacon_node.go [hunk 1]
```diff
@@ -94,12 +94,12 @@ func startNewBeaconNode(t *testing.T, config *end2EndConfig, beaconNodes []*beac
 	}
 
 	if err = waitForTextInFile(file, "Node started p2p server"); err != nil {
-		t.Fatal(err)
+		t.Fatal(fmt.Errorf("could not find multiaddr in node %d: %v", index, err))
```
Comment (nisdas) [inline endtoend/beacon_node.go]: use fatalf

File: endtoend/beacon_node.go [hunk 2]
```diff
@@ -94,12 +94,12 @@ func startNewBeaconNode(t *testing.T, config *end2EndConfig, beaconNodes []*beac
...
 	}
 
 	multiAddr, err := getMultiAddrFromLogFile(file.Name())
 	if err != nil {
-		t.Fatal(err)
+		t.Fatal(fmt.Errorf("could not get multiaddr in node %d: %v", index, err))
```
Comment (nisdas) [inline endtoend/beacon_node.go]: same here

File: endtoend/evaluators/finality.go [hunk 3]
```diff
@@ -20,7 +19,7 @@ var FinalizationOccurs = Evaluator{
 func finalizationOccurs(client eth.BeaconChainClient) error {
 	chainHead, err := client.GetChainHead(context.Background(), &ptypes.Empty{})
 	if err != nil {
-		return errors.Wrap(err, "failed to get chain head")
```
Comment (nisdas) [inline endtoend/evaluators/finality.go]: ? whats wrong with wrap

File: endtoend/beacon_node.go [hunk 4]
```diff
@@ -94,12 +95,12 @@ func startNewBeaconNode(t *testing.T, config *end2EndConfig, beaconNodes []*beac
 	}
 
 	if err = waitForTextInFile(file, "Node started p2p server"); err != nil {
-		t.Fatal(err)
+		t.Fatalf("could not find multiaddr for node %d, this means the node had issues starting: %v", index, err)
```
Comment (rauljordan) [inline endtoend/beacon_node.go]:
```suggestion
		t.Fatalf("Could not find multiaddr for node %d, this means the node had issues starting: %v", index, err)
```

File: endtoend/beacon_node.go [hunk 5]
```diff
@@ -94,12 +95,12 @@ func startNewBeaconNode(t *testing.T, config *end2EndConfig, beaconNodes []*beac
...
 	}
 
 	multiAddr, err := getMultiAddrFromLogFile(file.Name())
 	if err != nil {
-		t.Fatal(err)
+		t.Fatalf("could not get multiaddr fpr node %d: %v", index, err)
```
Comment (rauljordan) [inline endtoend/beacon_node.go]:
```suggestion
		t.Fatalf("Could not get multiaddr fpr node %d: %v", index, err)
```

File: endtoend/endtoend_test.go [hunk 6]
```diff
@@ -54,7 +55,7 @@ func runEndToEndTest(t *testing.T, config *end2EndConfig) {
 		t.Fatal(err)
 	}
 	if err := waitForTextInFile(beaconLogFile, "Sending genesis time notification"); err != nil {
-		t.Fatal(err)
+		t.Fatalf("failed to find genesis in logs, this means the chain did not start: %v", err)
```
Comment (rauljordan) [inline endtoend/endtoend_test.go]:
```suggestion
		t.Fatalf("Failed to find genesis in logs, this means the chain did not start: %v", err)
```

File: endtoend/endtoend_test.go [hunk 7]
```diff
@@ -77,14 +78,15 @@ func runEndToEndTest(t *testing.T, config *end2EndConfig) {
...
+
 		for _, evaluator := range config.evaluators {
 			// Only run if the policy says so.
 			if !evaluator.Policy(currentEpoch) {
 				continue
 			}
 			t.Run(fmt.Sprintf(evaluator.Name, currentEpoch), func(t *testing.T) {
 				if err := evaluator.Evaluation(beaconClient); err != nil {
-					t.Fatal(err)
+					t.Fatalf("evaluation failed for epoch %d: %v", currentEpoch, err)
```
Comment (rauljordan) [inline endtoend/endtoend_test.go]:
```suggestion
					t.Fatalf("Evaluation failed for epoch %d: %v", currentEpoch, err)
```

File: endtoend/eth1.go [hunk 8]
```diff
@@ -56,7 +56,7 @@ func startEth1(t *testing.T, tmpPath string) (common.Address, string, int) {
 	}
 
 	if err = waitForTextInFile(file, "Commit new mining work"); err != nil {
-		t.Fatal(err)
+		t.Fatalf("mining log not found, this means the eth1 chain had issues starting: %v", err)
```
Comment (rauljordan) [inline endtoend/eth1.go]:
```suggestion
		t.Fatalf("Mining log not found, this means the eth1 chain had issues starting: %v", err)
```

File: endtoend/validator.go [hunk 9]
```diff
@@ -109,7 +109,7 @@ func initializeValidators(
 	// Picked 20 for this as a "safe" number of blocks to mine so the deposits
 	// are detected.
 	if err := mineBlocks(web3, keystore, 20); err != nil {
-		t.Fatal(err)
+		t.Fatalf("failed to mine blocks %v", err)
```
Comment (rauljordan) [inline endtoend/validator.go]:
```suggestion
		t.Fatalf("Failed to mine blocks %v", err)
```

**Pull request merged**